src/commands/tail/mod.rs

```diff
@@ -152,9 +137,25 @@ async fn enable_tailing_start_heartbeat(
     }
 }
 
+async fn print_logs(req: Request<Body>) -> Result<Response<Body>, hyper::Error> {
+    match (req.method(), req.uri().path()) {
+        (&Method::POST, "/") => {
+            let whole_body = hyper::body::to_bytes(req.into_body()).await?;
+            println!("{}", str::from_utf8(&whole_body).unwrap());
```

Comment:
should probably replace the unwrap with a ? if at all possible, at least an expect